src/plugins/index.js

```diff
@@ -1,2 +1,3 @@
 import './provider'
-export * from './filters'
```

Comment:
New version of `eslint-config-standard` dependency had 2 unmet peer dependencies :
- `eslint-plugin-import`
- `eslint-plugin-node`

Then when running lint task, `eslint-plugin-import` detected an error:

```bash
/Users/npayot/dev/ngVue/src/plugins/index.js
2:15  error  No named exports found in module './filters'  import/export
```

Indeed, `export * from …;` does not export a default export from the imported module : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export